### PR TITLE
New version of front page with practical information for attendees

### PR DESCRIPTION
**Pull request by tjguk**
This should now be good to merge. Please comment &c. as you see fit.

### CODE REVIEW

File: pages/index.md
Comment (alexwlchan) [inline pages/index.md]: We don’t need the trailing space.

File: pages/index.md
Comment (alexwlchan) [inline pages/index.md]: “still have *a handful of*”?

And is this still true?

File: pages/index.md
Comment (alexwlchan) [inline pages/index.md]: “on the PyCon UK website” is redundant, surely; that’s where I am right now?

File: pages/index.md
Comment (alexwlchan) [inline pages/index.md]: Add a note here that you need to book?

File: pages/index.md
Comment (alexwlchan) [inline pages/index.md]: Do we want to mention this on the front page, given that the majority of readers won’t be able to attend?

File: pages/index.md
Comment (alexwlchan) [inline pages/index.md]: Missing link here.

File: pages/index.md
Comment (inglesp) [reply pages/index.md]: Yes.

File: pages/index.md
Comment (inglesp) [inline pages/index.md]: We can link to http://2017.pyconuk.org/sessions/workshops/sprint/ here.

File: pages/index.md
Comment (inglesp) [reply pages/index.md]: Yes.

File: pages/index.md
Comment (inglesp) [inline pages/index.md:21]: We can offer accommodation until Wednesday, and we're still getting the odd message about accommodation, so could you leave this up here somewhere?

File: pages/index.md
Comment (inglesp) [inline pages/index.md]: Change "talk details" to "ticket details"?

File: pages/index.md
Comment (inglesp) [reply pages/index.md]: I think we should include it, even though it's not relevant to most people, because a surprising number of people are bad at searching for emails with information.  But we should mention that it's invite-only, and tell speakers to get in touch if they've not received an invite.

File: pages/index.md
Comment (inglesp) [inline pages/index.md]: Add "Ticketholders will receive an invitation to join in the week before the conference", otherwise we'll get people asking how to join.  (I mean, we'll still get people asking how to join...)

File: pages/index.md
Comment (inglesp) [inline pages/index.md]: Some of these (Picademy and Django Girls) require signing up in advance, and are full.  I'd like to rewrite this as follows:

### Other events at PyCon UK

* *[PyData](/pydata/)* (Thursday to Sunday) Talks and workshops about data science, put on in collaboration with [PyData London](https://london.pydata.org/).  Access is included in the price of the conference ticket.

* *[Django Girls](https://djangogirls.org/pyconuk2017/)* (Thursday) A beginner-friendly Python and Django workshop designed to inspire women to fall in love with programming.  Applications are closed.  If you have any questions, please contact the Django Girls organisers by email: pyconuk@djangogirls.org.

* *[Picademy](https://www.raspberrypi.org/training/picademy/)* (Thursday and Friday) A professional development programme for teachers, organised by the [Raspberry Pi Foundation](https://www.raspberrypi.org/).  Applications are closed.  If you have any questions, please [contact the Raspberry Pi Foundation directly](https://www.raspberrypi.org/contact/).

* *[Talks for teachers](http://2017.pyconuk.org/schedule/#saturday)* (Saturday) A day of talks in Room D about Python and education.  Non-teachers are very welcome!

* *[Code Club](/sessions/workshops/code-club/)* (Saturday) Coding workshops for kids aged under 13.  Led by [Code Club](https://www.codeclub.org.uk/) champions and volunteers, we'll be running activities for young coders' using Python to create games and more. There will also be a special session on programming micro:bits and Adafruit's Circuit Playground Express boards - not to be missed!  [Tickets cost £5 and are still on sale](https://hq.pyconuk.org/children/orders/new/)!

* *[Raspberry Jam](/sessions/workshops/raspberry-jam/)* (Saturday) More coding workshops, for young people aged over 13, or anyone looking to do more advanced projects.  The [Raspberry Pi Foundation](https://www.raspberrypi.org/) will be running a Raspberry Jam - and parents and guardians can join in too! We'll be running a variety of digital making activities using Python on the Raspberry Pi.  [Tickets cost £5 and are still on sale](https://hq.pyconuk.org/children/orders/new/)!

* *[Trans*Code](/transcode/)* (Monday) A day-long hackathon to address issues facing the trans* and nonbinary community. The day is open to all trans* and nonbinary people and allies, and we welcome participants from all skill levels and backgrounds.

**Review by inglesp — CHANGES_REQUESTED** [review]
I think this is great, and I think you've saved me the job of writing the last pre-conference email!

I've made some suggested improvements in the comments.

File: pages/index.md
Comment (tjguk) [reply pages/index.md]: Fixed

File: pages/index.md
Comment (tjguk) [reply pages/index.md]: Just a bit more compact with a lot else on the page. Yes; Carrie Anne says they can take 100 -- we're at 77

File: pages/index.md
Comment (tjguk) [reply pages/index.md]: I know. I was akwardly trying to find somewhere to link to and I hate "<>here</>" links. I've reworded it.

File: pages/index.md
Comment (tjguk) [reply pages/index.md]: Yes; I'll rework this slightly.

File: pages/index.md
Comment (tjguk) [reply pages/index.md]: It's a bit 50/50 but my aim was for people to have a one-stop-shop of where to look. And there are more than 100 people going, I think. I didn't think it was too egregious.

File: pages/index.md
Comment (tjguk) [reply pages/index.md]: Thanks; we seem to have some confusion over who's actually organising it, so we don't have anything to link to at the moment. If nothing comes through tonight, I'll ditch the reference.

File: pages/index.md
Comment (tjguk) [reply pages/index.md:21]: Added to the Venue section

File: pages/index.md
Comment (tjguk) [reply pages/index.md]: Done

File: pages/index.md
Comment (tjguk) [reply pages/index.md]: Added

File: pages/index.md
Comment (tjguk) [reply pages/index.md]: Thanks; I really wasn't sure what to put there. I'll cut-and-paste yours in.

File: pages/dojo.md
Comment (ntoll) [inline pages/dojo.md]: Except in this case we don't. Perhaps change to:

"We usually choose a challenge, but in this case we're going write bots to play each other at this Python game: https://robotgame.net/"

File: pages/dojo.md
Comment (ntoll) [inline pages/dojo.md:24]: I thought people were just going to turn up (rather than sign up)..? I.e. the whole point of choosing the robot game was to accommodate plenty of teams. (I may be misremembering).

File: pages/dojo.md
Comment (tjguk) [reply pages/dojo.md]: Done

File: pages/dojo.md
Comment (tjguk) [reply pages/dojo.md:24]: I defer to @inglesp but I think the point is that we need *some* idea of numbers (and perhaps a cap) so that we don't overflow the venue and have too much / too little pizza.

Comment (inglesp) [reply pages/dojo.md:24]: We need to give the venue an idea of numbers for catering purposes, otherwise we'll either (a) have a repeat of potatogate, or (b) be on the hook for 1000 pizzas.

Comment (ntoll) [reply pages/dojo.md:24]: OK... so given the nature of the challenge, we need 8 teams. If we have 5 as the maximum number for a team that gives us an upper limit of 40 attendees. In the past, we're just done a first come, first served thing and closed the door when 40 people have walked in. Does that help?

Comment (inglesp) [reply pages/dojo.md:24]: I'm surprised to see a limit of 40 -- I thought the whole point of choosing the task you chose was to avoid having a limit.  Can we discuss in Slack?

**Review by inglesp — APPROVED** [review]

File: pages/dojo.md
Comment (meatballs) [inline pages/dojo.md]: typo: going to write bots

**Review by meatballs — CHANGES_REQUESTED** [review]

File: pages/dojo.md
Comment (inglesp) [reply pages/dojo.md]: What's the typo?

**Review by meatballs — APPROVED** [review]

File: pages/index.md
Comment (alexwlchan) [inline pages/index.md]: double parens here are breaking the build

**Comment by inglesp** [comment]
Please can we have no more commits to this PR?  If we need to make changes, please wait until this is merged, and make a new PR?

File: pages/index.md
Comment (meatballs) [inline pages/index.md]: url should be /sessions/talks/lightning-talks

**Comment by ntoll** [comment]
Hurrah!

**Pull request merged**
